lxd/api_project.go

```diff
@@ -235,6 +239,8 @@ func projectPut(d *Daemon, r *http.Request) response.Response {
 		project.Description,
 		project.Config["features.images"],
 		project.Config["features.profiles"],
+		project.Config["limits.containers"],
+		project.Config["limits.virtual-machines"],
```

Comment:
Same comment as below, just pass `project.Config` once and that should cover all 4 keys.